bundle/language_en.properties

```diff
@@ -1221,8 +1227,9 @@ angal.opd.reattendance.btn
 angal.opd.reattendance.txt                                                                             = Re-Attendance
 angal.opd.referral.txt                                                                                 = Referral
 angal.opd.referralto.txt                                                                               = Referred To
+angal.opd.searchbycodespressenter.border                                                               = Search by codes (press ENTER)
```

Comment:
```suggestion
angal.opd.searchbycodespressenter.border                                                               = Search by Code (press ENTER)
```

Reply:
The plural was intentional (we have three different codes)
Is it bad English?

Reply:
The search only uses one of the fields so one is searching on single code.